packages/language-picker/src/style.scss

```diff
@@ -0,0 +1,47 @@
+@import '~@wordpress/base-styles/breakpoints';
+@import '~@wordpress/base-styles/mixins';
+@import '~@wordpress/base-styles/variables';
+
+.language-picker__language-group {
+	width: 100%;
+}
+
+.language-picker__language-group > span {
```

Comment:
Nit: I wonder if this `span` can have its own `className` that will always be there; so we could target it by class name and not by element name.